### PR TITLE
custom cookie

### PR DESCRIPTION
**Pull request by LvThxMax**
custom cookie support, api is similar to CookiSessionBackend.

### CODE REVIEW

File: src/session.rs [hunk 1]
```diff
@@ -79,6 +80,11 @@ impl RedisSessionBackend {
             ttl: "7200".to_owned(),
             addr: RedisActor::start(addr),
             name: "actix-session".to_owned(),
+            path: "/".to_owned(),
+            domain: None,
+            secure: false,
+            max_age: None,
```
Comment (fafhrd91) [inline src/session.rs]: Let’s set some value by default

Comment (yinyanlv) [reply src/session.rs]: max_age: Duration::days(90)  ?

Comment (yinyanlv) [reply src/session.rs]: max_age: Some(Duration::day(90)) ?

Comment (lvthxmax) [reply src/session.rs]: @fafhrd91  max_age: Some(Duration::days(7)) ?

Comment (fafhrd91) [reply src/session.rs]: Yes

Comment (lvthxmax) [reply src/session.rs]: @fafhrd91  max-age default value has been setted

**Review by fafhrd91 — APPROVED** [review]

**Comment by fafhrd91** [comment]
thanks!

**Pull request merged**